### PR TITLE
update upgradable dependencies

### PR DESCRIPTION
**Pull request by krystophv**
within reason

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,8 +18,8 @@
   },
   "dependencies": {
     "async": "1.5.2",
-    "babyparse": "0.4.3",
-    "blueimp-md5": "2.1.0",
+    "babyparse": "0.4.5",
```
Comment (jebeck) [inline package.json]: So I think I asked this, but I'll ask again. `babyparse` is the CSV parsing library, right @krystophv? Which means we use it for CareLink ingestion, so as I dev updating this it'd be good to at least do a smoke test of CareLink uploading as part of developer testing.

**Comment by jebeck** [comment]
Is there a commit you've forgotten to push here @krystophv?

When attempting to run locally:
```
ERROR in ./~/babyparse/babyparse.js
Module not found: Error: Cannot resolve module 'fs' in /Volumes/Tidepool/chrome-uploader/node_modules/babyparse
 @ ./~/babyparse/babyparse.js 41:16-29
```

**Comment by krystophv** [comment]
@jebeck 
Re: babyparse update - Did a basic upload and view test with success. 
Re: `fs` error - no, I just forgot to clear node_modules and start fresh before doing final pass testing locally. I must have had a more recent version of something that allowed that to work properly. I added back in the fix.

**Comment by jebeck** [comment]
:ship:

**Comment by gniezen** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;o9yH83Bk&#x2F;2-ux-0-dev-2-tst-1-uploader-device-onetouch-verioiq"> &lt;ux:0, dev:2, tst:1&gt; [uploader] [device] OneTouch VerioIQ</a>

**Pull request merged**